src/modify.jl

```diff
@@ -40,3 +40,27 @@ function rename(ta::TimeArray, colnames::Vector{Symbol})
 end
 
 rename(ta::TimeArray, colnames::Symbol) = rename(ta, [colnames])
+
+function rename(ta::TimeArray, args...)
```

Comment:
IIRC, this is legit:
`function rename(ta::TimeArray, args::Pair{Symbol,Symbol}...)`